front/client/components/Article.vue

```diff
@@ -10,8 +10,25 @@
       </div>
       <!-- <div class="article-description" v-html="getCategoryDescription"></div> -->
       <!-- TODO куда то пропал category description - разобраться -->
-      <div class="article-description" v-html="getCategoryDescription"></div>
-      <article-content v-bind:contentItem = "getCategoryItems" />
+      <!-- <div class="article-description" v-html="getCategoryDescription"></div> -->
+      <div v-for="(value, index) in getCategoryItems" :key="index">
```

Comment:
Is it necessary to use wraping `div`? You can use `v-for` on `<article-content>` tag itself